js_hw_4/task_01.js

```diff
@@ -0,0 +1,30 @@
+// 1. Создайте функцию getEmployeeInfo()
+// 2. В функции создайте массив имен сотрудников 5шт (Имена придумать самим)
+// 3. В функции создайте массив зарплат сотрудников 5 шт(цифры придумать самим)
+// 4. Функция должна принимать 1 аргумент - имя сотрудника
+// 5. Функция должна возвращать новый массив, где первый элемент - имя сотрудника, второй - его зарплата
+// 6. Для поиска ответа функции нужно найти индекс сотрудника в массиве имен. Зарплату взять с ТЕМ ЖЕ индексом что и имя
+// 7. Для возврата из функции создайте массив, методом .push поместите в него имя и зарплату, и через return верните созданный массив
+// 8. Если такое имя сотрудника в массиве не найдется - вернуть null
+
+function getEmployeeInfo(name) {
+    if(name!== undefined && name!== null && name.length && !+name){
```

Comment:
```suggestion
    if(!name || !name.trim().length) { /* trow error */}
```

Reply:
@VladimirLQA your solution will throw "TypeError: name.trim is not a function" when the number is provided in input

Reply:
So you can add check on typeof for strings + empty string with logical `and` operator